interrogatio/widgets/wizard.py

```diff
@@ -55,6 +56,9 @@ def __init__(  # noqa: CCR001
 
         self.buttons = [self.next_btn, self.cancel_btn]
 
+        if self.summary and skip_questions:
```

Comment:
IMO if there is no summary it should go to the latest question or if validation of one of the question fails it should stop on such question.